.github/workflows/psv_pipelines.yml

```diff
@@ -273,3 +273,44 @@ jobs:
     - name: Commit checker script. Verify commit text
       run: scripts/misc/commit_checker.sh
       shell: bash
+
+  psv-formatting-checker:
+    name: PSV.Clang.Format.Checker
+    runs-on: ubuntu-22.04
+    env:
+      CLANG_FORMAT_FILE: "clang-format.diff"
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v4
+    - name: Setup environment
+      run: |
+        set +x
+        set -e
+        sudo apt-get update
+        sudo apt-get install -y wget libedit2 libtinfo5 python2
+        alias python=python2
+        mkdir _os_deps
+        cd _os_deps
+        wget https://apt.llvm.org/bionic/pool/main/l/llvm-toolchain-6.0/clang-format-6.0_6.0.1~svn334776-1~exp1~20190309042703.125_amd64.deb
+        wget https://apt.llvm.org/bionic/pool/main/l/llvm-toolchain-6.0/libllvm6.0_6.0.1~svn334776-1~exp1~20190309042703.125_amd64.deb
+        wget http://old-releases.ubuntu.com/ubuntu/pool/main/libf/libffi/libffi6_3.2.1-9_amd64.deb
+        sudo dpkg -i libffi6_3.2.1-9_amd64.deb
+        sudo dpkg -i libllvm6.0_6.0.1~svn334776-1~exp1~20190309042703.125_amd64.deb
+        sudo dpkg -i --force-depends clang-format-6.0_6.0.1~svn334776-1~exp1~20190309042703.125_amd64.deb
+        cd ..
+      shell: bash
+    - name: "Clang format checker script"
+      run: ./scripts/misc/clang_format_ci.sh
+      shell: bash
+    - name: Store formatting check results
+      uses: actions/upload-artifact@v4
+      with:
+        name: clang-format-diff
+        path: ${{ env.CLANG_FORMAT_FILE }}
+    - name: Verify check result
+      run: |
+        set +x
+        if [ -s ${CLANG_FORMAT_FILE} ] ; then
+          exit 1
```

Comment:
Put the message here with explanation that the link to the patch is in the previous step